app/src/main/java/org/jellyfin/androidtv/details/ItemListActivity.java

```diff
@@ -675,13 +675,13 @@ public void onClick(DialogInterface dialog, int which) {
                         } else {
                             new AlertDialog.Builder(mActivity)
                                     .setTitle(R.string.lbl_delete)
-                                    .setMessage("This will PERMANENTLY DELETE " + mBaseItem.getName() + " from your library.  Are you VERY sure?")
-                                    .setPositiveButton("Delete", new DialogInterface.OnClickListener() {
+                                    .setMessage(R.string.warning_0 + mBaseItem.getName() + R.string.warning_1)
```

Comment:
Using a string template is preferred for strings that contain dynamic elements. An example in this case: `This will PERMANENTLY DELETE %1$s from your library.  Are you VERY sure?`. The value can then be passed to the `getString` function as argument.

[Android documentation](https://developer.android.com/guide/topics/resources/string-resource.html#FormattingAndStyling)

Reply:
Thanks for the documentation and clear guide. I believe I have resolved the issues, so kindly have a look